db/api/v1/db/assetDefinitions.db.js

```diff
@@ -124,7 +136,227 @@ const create = async (assetDefinition) => {
     }
 };
 
+/**
+ * Gets all properties associated with an asset type using the asset type's ID
+ * 
+ * @param {Number} assetTypeId ID of the asset type whose properties are being queried
+ */
+const findPropertiesByAssetTypeId = async(assetTypeId) => {
```

Comment:
I believe there is already an endpoint in `dev`, this code can probably be converted to use that one

Reply:
The one in dev accesses the asset_property table. This one accesses the property table.